examples/complicated/validators.go

```diff
@@ -170,26 +160,22 @@ func (r User) Validate() error {
 		errs.AddFieldf("Dict", "less items than 2")
 	}
 	for kDict, vDict := range r.Dict {
-		_ = kDict
-		_ = vDict
 		if utf8.RuneCountInString(string(kDict)) > 64 {
-			errs.AddFieldf(fmt.Sprintf("Dict"+".%v", kDict), "longer than 64 chars")
+			errs.AddFieldf(fmt.Sprintf("Dict"+".key[%v]", kDict), "longer than 64 chars")
 		}
 		if vDict < -35 {
-			errs.AddFieldf(fmt.Sprintf("Dict"+".%v", vDict), "less than -35")
+			errs.AddFieldf(fmt.Sprintf("Dict"+".%v", kDict), "less than -35")
 		}
 		if vDict > 34 {
-			errs.AddFieldf(fmt.Sprintf("Dict"+".%v", vDict), "more than 34")
+			errs.AddFieldf(fmt.Sprintf("Dict"+".%v", kDict), "more than 34")
 		}
 	}
 	for kDictDogs, vDictDogs := range r.DictDogs {
-		_ = kDictDogs
-		_ = vDictDogs
 		if err := vDictDogs.ValidateOptional(); err != nil {
-			errs.AddField(fmt.Sprintf("DictDogs"+".%v", vDictDogs), err)
+			errs.AddField(fmt.Sprintf("DictDogs"+".%v", kDictDogs), err)
```

Comment:
Are you sure it will be better to see key instead of value when validating value ? In this case seems i need to repeat test and find value manually by debugging.